ocs_ci/ocs/ui/base_ui.py

```diff
@@ -485,71 +497,78 @@ class PageNavigator(BaseUI):
 
     """
 
-    def __init__(self, driver):
-        super().__init__(driver)
-        self.ocp_version = get_ocp_version()
-        self.ocp_version_full = version.get_semantic_ocp_version_from_config()
-        self.page_nav = locators[self.ocp_version]["page"]
-        self.ocs_version_semantic = version.get_semantic_ocs_version_from_config()
-        self.ocp_version_semantic = version.get_semantic_ocp_version_from_config()
-        self.running_ocp_semantic_version = version.get_semantic_ocp_running_version()
-        self.operator_name = (
-            ODF_OPERATOR
-            if self.ocs_version_semantic >= version.VERSION_4_9
-            else OCS_OPERATOR
-        )
-        if Version.coerce(self.ocp_version) >= Version.coerce("4.8"):
-            self.generic_locators = locators[self.ocp_version]["generic"]
-
-        if config.DEPLOYMENT.get("local_storage", False):
-            self.storage_class = "localblock_sc"
-        elif config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM:
-            if self.ocs_version_semantic <= version.VERSION_4_12:
-                self.storage_class = "thin_sc"
-            else:
-                self.storage_class = "thin-csi_sc"
-        elif config.ENV_DATA["platform"].lower() == constants.AWS_PLATFORM:
-            aws_sc = config.DEPLOYMENT.get("customized_deployment_storage_class")
-            if aws_sc == "gp3-csi":
-                self.storage_class = "gp3-csi_sc"
-            elif aws_sc == "gp2-csi":
-                self.storage_class = "gp2-csi_sc"
-            else:
-                if self.running_ocp_semantic_version >= version.VERSION_4_12:
-                    self.storage_class = "gp2-csi_sc"
-                else:
-                    self.storage_class = "gp2_sc"
-        elif config.ENV_DATA["platform"].lower() == constants.AZURE_PLATFORM:
-            if self.ocp_version_semantic >= version.VERSION_4_11:
-                self.storage_class = "managed-csi_sc"
-            else:
-                self.storage_class = "managed-premium_sc"
-        elif config.ENV_DATA["platform"].lower() == constants.GCP_PLATFORM:
-            if self.ocs_version_semantic < version.VERSION_4_12:
-                self.storage_class = "standard_sc"
+    ocp_version = get_ocp_version()
+    running_ocp_semantic_version = version.get_semantic_ocp_running_version()
+    ocp_version_full = version.get_semantic_ocp_version_from_config()
+    ocs_version_semantic = version.get_semantic_ocs_version_from_config()
+    ocp_version_semantic = version.get_semantic_ocp_version_from_config()
```

Comment:
Why did you remove `self.` from ocp/ocs versions?

Reply:
these variables are static, they are the same for entire test run. Calling them in constructor each instantiation of each child class is redundant, so they had been passed as a class variables and we still can use `instance.property`

class Student:
    # Class variable
    school_name = 'ABC School '

    # constructor
    def __init__(self, name, roll_no):
        self.name = name
        self.roll_no = roll_no

# create Object
s1 = Student('Emma', 10)

# access using object reference
print(s1.school_name) 
--------
ABC School